docker/docker/webapp/Dockerfile.nginx

```diff
@@ -0,0 +1,22 @@
+# source webapp/.env
+# docker build --build-arg REACT_APP_BACKEND_URI=$REACT_APP_BACKEND_URI -f docker/webapp/Dockerfile.nginx -t chat-copilot-webapp-nginx .
+
+# builder
+FROM node:lts-alpine as builder
+ARG REACT_APP_BACKEND_URI
+# ENV REACT_APP_BACKEND_URI $REACT_APP_BACKEND_URI
```

Comment:
Any reason for leaving this comment here?

Reply:
Hello, I still encountered the problem I told you yesterday, I ran the container through these two commands, the front page is white screen.

`docker build --build-arg REACT_APP_BACKEND_URI=$REACT_APP_BACKEND_URI -f docker/webapp/Dockerfile.nginx -t chat-copilot-webapp-nginx . `
or
`docker-compose -f .\docker\docker-compose.yaml up -d`

Reply:
the leftover files are cleaned up